<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>rncbc aka Rui Nuno Capela</author>
 <comment>qtractor - An Audio/MIDI multi-track sequencer.

   Copyright (C) 2010, rncbc aka Rui Nuno Capela. All rights reserved.

   This program is free software; you can redistribute it and/or
   modify it under the terms of the GNU General Public License
   as published by the Free Software Foundation; either version 2
   of the License, or (at your option) any later version.

   This program is distributed in the hope that it will be useful,
   but WITHOUT ANY WARRANTY; without even the implied warranty of
   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
   GNU General Public License for more details.

   You should have received a copy of the GNU General Public License along
   with this program; if not, write to the Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.

 </comment>
 <class>qtractorMidiControlObserverForm</class>
 <widget class="QDialog" name="qtractorMidiControlObserverForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>264</width>
    <height>161</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MIDI Controller - Qtractor</string>
  </property>
  <property name="windowIcon">
   <iconset>:/images/itemControllers.png</iconset>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <layout class="QGridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="ControlTypeTextLabel">
       <property name="text">
        <string>&amp;Type:</string>
       </property>
       <property name="buddy">
        <cstring>ControlTypeComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QComboBox" name="ControlTypeComboBox"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="ChannelTextLabel">
       <property name="text">
        <string>Cha&amp;nnel:</string>
       </property>
       <property name="buddy">
        <cstring>ChannelSpinBox</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="ChannelSpinBox">
       <property name="specialValueText">
        <string>(Any)</string>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>16</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="ParamTextLabel">
       <property name="text">
        <string>&amp;Parameter:</string>
       </property>
       <property name="buddy">
        <cstring>ParamComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QComboBox" name="ParamComboBox"/>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QCheckBox" name="LogarithmicCheckBox">
       <property name="text">
        <string>&amp;Logarithmic</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QCheckBox" name="FeedbackCheckBox">
       <property name="text">
        <string>&amp;Feedback</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="DialogButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="4" margin="4"/>
 <tabstops>
  <tabstop>ControlTypeComboBox</tabstop>
  <tabstop>ChannelSpinBox</tabstop>
  <tabstop>ParamComboBox</tabstop>
  <tabstop>FeedbackCheckBox</tabstop>
  <tabstop>LogarithmicCheckBox</tabstop>
  <tabstop>DialogButtonBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
